### ISSUE TITLE
First mission: Contributors page

### ISSUE THREAD

**Issue by coni2k** [FreeCodeCamp/how-to-contribute-to-open-source#29]
@shubheksha, thanks for drawing attention to this, appreciated.

As a maintainer, wrote an article about how could we lower the barrier for newcomers, could be useful for you?
https://medium.com/@forCrowd/first-mission-contributors-page-df24e6e70705#.2v2g0no29

**Comment by shubheksha** [comment]
@coni2k, thank you - yes, certainly! We'll be adding a list of articles for maintainers to make their project more beginner-friendly as well. 😃

**Comment by coni2k** [comment]
@shubheksha, sounds great, I will follow the news :)

And meanwhile, since this tutorial kind of approach helped us to reach many new members (15+ new PRs, 20+ new forks in couple of months), I would like to work on extending it with new missions. If I can create something solid, I will post it here to let you know. Thanks again!

**Comment by AbiSadhasivam** [comment]
@coni2k  :  Is it still open? If yes, Can I take it up?

**Comment by coni2k** [comment]
@AbiSadhasivam Yes, you can follow the article and send your PR. If you will have any questions and/or anything unclear, please feel free to send a message Good luck! :)

**Comment by ravk1234** [comment]
Is it still open?

**Comment by coni2k** [comment]
@ravk1234 yes, it's still open. Please feel free if you want to send a PR
